docs/content/modeling/migrating-relations.mdx

```diff
@@ -0,0 +1,310 @@
+
+## Before You Start
+
+In order to understand this guide correctly you must be familiar with some <ProductConcept /> and know how to develop the things that we will list below.
+
+<details>
+<summary>
+
+You will start with the _<ProductConcept section="what-is-an-authorization-model" linkName="authorization model" />_ below,
+it represents a `document` _<ProductConcept section="what-is-a-type" linkName="type" />_ that can have users assigned as editor relation. Editor can edit document.
+
+</summary>
+
+<AuthzModelSnippetViewer
+  configuration={{
+    type_definitions: [
+      {
+        type: 'document',
+        relations: {
+          editor: {
+            this: {},
+          },
+          can_edit: {
+            computedUserset: {
+              object: '',
+              relation: 'editor',
+            },
+          },
+        },
+      },
+      {
+        type: 'user',
+      },
+    ],
+  }}
+/>
+
+There are existing <ProductConcept section="what-is-a-relationship-tuple" linkName="relationship tuples" /> associated with editor relation.
+
+<RelationshipTuplesViewer
+  relationshipTuples={[
+    {
+      user: 'user:anne',
+      relation: 'editor',
+      object: 'document:roadmap',
+    },
+    {
+      user: 'user:charles',
+      relation: 'editor',
+      object: 'document:roadmap',
+    },
+  ]}
+/>
+
+<hr />
+
+In addition, you will need to know the following:
+
+### <ProductName format={ProductNameFormat.ShortForm} /> Concepts
+
+- A <ProductConcept section="what-is-a-type" linkName="Type" />: a class of objects that have similar characteristics
+- A <ProductConcept section="what-is-a-user" linkName="User" />: an entity in the system that can be related to an object
+- A <ProductConcept section="what-is-a-relation" linkName="Relation" />: is a string defined in the type definition of an authorization model that defines the possibility of a relationship between an object of the same type as the type definition and a user in the system
+- An <ProductConcept section="what-is-an-object" linkName="Object" />: represents an entity in the system. Users' relationships to it can be define through relationship tuples and the authorization model
+- A <ProductConcept section="what-is-a-relationship-tuple" linkName="Relationship Tuple" />: a grouping consisting of a user, a relation and an object stored in <ProductName format={ProductNameFormat.ShortForm} />
+- [Intersection Operator](../configuration-language.mdx#the-intersection-operator): the intersection operator can be used to indicate a relationship exists if the user is in all the sets of users
+
+</details>
+
+## Step By Step
+
+Assume that you want to migrate to the following model:
```

Comment:
```suggestion
## Create a backwards compatible model

To avoid service disruption, you will create a backwards compatible model. The backwards compatible model ensures the existing relationship tuple will still work.
```